src/backend/utils.ts

```diff
@@ -394,6 +394,8 @@ async function errorHandler({
   const deletedFolderMsg = 'appears to be deleted'
   const expiredCredentials = 'No saved credentials'
   const legendaryRegex = /legendary.*\.py/
+  // this message appears on macOS when no Crossover was found in the system but its a false alarm
+  const ignoreMessage = 'IndexError: list index out of range'
```

Comment:
```suggestion
  const ignoreCrossoverMessage = 'IndexError: list index out of range'
```

just for context when used later